PAWS/Study Information/EnrollmentGroup.swift

```diff
@@ -0,0 +1,41 @@
+//
+// This source file is part of the PAWS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import Spezi
+import SpeziAccount
+import SpeziFirebaseAccount
+import SwiftUI
+
+
+@Observable
+class EnrollmentGroup: Module, EnvironmentAccessible {
+    @ObservationIgnored @Dependency var accountConfiguration: FirebaseAccountConfiguration?
```

Comment:
Is this what you meant by adding a dependency toward Spezi configuration of Firebase? @PSchmiedmayer

Reply:
This is the `@Dependency` module that I was referring to, you can learn more about this at https://swiftpackageindex.com/StanfordSpezi/SpeziFirebase/main/documentation/spezifirebaseconfiguration
```suggestion
    @ObservationIgnored @Dependency private var configureFirebaseApp: ConfigureFirebaseApp
```